bootstrap-salt.sh

```diff
@@ -1587,6 +1587,57 @@ movefile() {
 
 
 #---  FUNCTION  -------------------------------------------------------------------------------------------------------
+#          NAME:  linkfile
+#   DESCRIPTION:  Simple function to create symlinks. Overrides if asked. Accepts globs.
+#----------------------------------------------------------------------------------------------------------------------
+linkfile() {
```

Comment:
Please prefix this with `__`. The only intent is to "tell" someone reading that it's an internal function, there's no other gain from that.